python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -124,8 +126,33 @@ def build(ctx: click.Context, **kwargs: Any) -> None:
     _unhandledShow(show, "build")
 
 
+def start_coverage(coverage_packages: Tuple) -> Any:
+    print("Coverage turned ON!", file=sys.stderr)
+    if coverage_packages:
+        print("Coverage limited to: " + ",".join(coverage_packages), file=sys.stderr)
+
+    import coverage
+
+    coveragerc = os.path.join(pathlib.Path(__file__).parent.resolve(), "coveragerc")
+    cov = coverage.Coverage(
+        branch=True, concurrency="multiprocessing", config_file=coveragerc, source_pkgs=coverage_packages
+    )
+    cov.load()
```

Comment:
It's this line that is loading previous coverage data. You need to think about what that means. It might be a useful feature if we understood how it worked.

Reply:
Thanks for catching that, that's from the coverage docs that has that in there I think.  I'd rather just not have it load previous results as that is super confusing, at least it was for me.

Reply:
Okay. You might have to add it to the extra files included when the PyPI package is built since it's not a .py file so won't be found by default (like the py.typed file isn't). See the pyproject.toml file.

Reply:
I deleted the file and just am programmatically generating it from the code now since you said we may be overriding it, so I shouldn't need to add it anywhere else, it's just a string in there

Reply:
Lines 130-137